app/scss/_colors.scss

```diff
@@ -0,0 +1,8 @@
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// Colors
+//
+// A collection of the color variables for DIM.
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+$black: #000;
+$white: #fff;
```

Comment:
This is totally unnecessary. CSS already has color definitions for `black` and `white`. Please don't add this level of indirection.